module/VuFind/src/VuFind/ILS/Logic/RenewalsHelper.php

```diff
@@ -0,0 +1,171 @@
+<?php
+
+/**
+ * VuFind Helper - Renewals Support Methods
+ *
+ * PHP version 8
+ *
+ * Copyright (C) Villanova University 2010.
+ *
+ * This program is free software; you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 2,
+ * as published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program; if not, see
+ * <https://www.gnu.org/licenses/>.
+ *
+ * @category VuFind
+ * @package  ILS_Logic
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+
+namespace VuFind\ILS\Logic;
+
+use Laminas\Mvc\Plugin\FlashMessenger\FlashMessenger;
+use Laminas\Stdlib\Parameters;
+use VuFind\ILS\Connection;
+use VuFind\Validator\CsrfInterface;
+
+use function is_array;
+
+/**
+ * Helper to perform renewal-related actions
+ *
+ * @category VuFind
+ * @package  ILS_Logic
+ * @author   Demian Katz <demian.katz@villanova.edu>
+ * @license  http://opensource.org/licenses/gpl-2.0.php GNU General Public License
+ * @link     https://vufind.org Main Page
+ */
+class RenewalsHelper
+{
+    /**
+     * Update ILS details with renewal-specific information, if appropriate.
+     *
+     * @param Connection $catalog     ILS connection object
+     * @param array      $ilsDetails  Transaction details from ILS driver's
+     * getMyTransactions() method
+     * @param array|bool $renewStatus Renewal settings from ILS driver's
+     * checkFunction() method
```

Comment:
With our new, longer line length limit, we can safely unwrap these lines:

```suggestion
     * @param array      $ilsDetails  Transaction details from ILS driver's getMyTransactions() method
     * @param array|bool $renewStatus Renewal settings from ILS driver's checkFunction() method
```